### PR TITLE
Add draw.io information

### PR DESCRIPTION
**Pull request by ItayZiv**
Add information on how to save draw.io files as svg and still have them be editable via metadata and add style-guide info.

### CODE REVIEW

File: source/docs/contributing/drawio-saving-instructions.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Draw.io Saving Instructions
+===========================
+
+This is a explanation of how to save diagrams made in ``draw.io`` (Otherwise known as ``diagrams.net``) as a ``.svg`` file that contains metadata to still be editable in ``draw.io``.
+
+.. image:: images/drawio-example.svg
```
Comment (Daltz333) [inline source/docs/contributing/drawio-saving-instructions.rst:6]: Where is there 3 SVGs?

File: source/index.rst [hunk 2]
```diff
@@ -90,6 +90,7 @@ Welcome to the *FIRST*\ |reg| Robotics Competition Control System Documentation!
    docs/contributing/style-guide
    docs/contributing/build-instructions
    docs/contributing/top-contributors
+   docs/contributing/drawio-saving-instructions
```
Comment (Daltz333) [inline source/index.rst:93]: Put above top-contributors

File: source/docs/contributing/drawio-saving-instructions.rst (hunk 1 above)
Comment (ItayZiv) [reply source/docs/contributing/drawio-saving-instructions.rst:6]: Those were just to verify RTD / PDF builds work properly

File: source/docs/contributing/frc-docs/drawio-saving-instructions.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+===========================
+
+.. warning:: Make sure you don't modify any file that is in a ``diagrams`` folder, or ends in ``.drawio.svg`` in any program other than draw.io, otherwise you might risk breaking the metadata of the file, making it uneditable.
+
+Draw.io (also known as `diagrams.net <https://app.diagrams.net/>`__) are supported when saved as ``svg`` files, with embedded XML metadata for the draw.io source file (normally stored as ``.drawio``).
+This allows these images to both act as source files for the diagrams that can be edited in the future, and be rendered as normal ``svg`` files.
+
+There are a few methods to save a diagram with the embedded metadata, but using the export menu is preferred because it allows us to embed any images in the diagram, otherwise they might not render properly on the docs.
+
+
```
Comment (AustinShalit) [inline source/docs/contributing/frc-docs/drawio-saving-instructions.rst:11]:
```suggestion
```

File: source/docs/contributing/frc-docs/style-guide.rst [hunk 4]
```diff
@@ -167,6 +167,27 @@ Simply use them as you would with any other image.
...
+
+.. code-block:: text
+
+   .. images:: diagrams/diagram-1.drawio.svg
+
+Draw.io Files
+~~~~~~~~~~~~~
+
+Draw.io files follow almost the same naming scheme as normal images. To keep track of files that have the embedded ``.drawio`` metadata, append a ``.drawio`` to the end of the file name, before the extension, meaning the name of the file should be ``document-title-1.drawio.svg`` and so on.
+Additionally, diagrams should be stored in the document directory in a sub-folder named ``diagrams``.
```
Comment (AustinShalit) [inline source/docs/contributing/frc-docs/style-guide.rst:185]:
```suggestion
Draw.io files follow almost the same naming scheme as normal images. To keep track of files that have the embedded ``.drawio`` metadata, append a ``.drawio`` to the end of the file name, before the extension, meaning the name of the file should be ``document-title-1.drawio.svg`` and so on. Additionally, diagrams should be stored in the document directory in a sub-folder named ``diagrams``.
```

**Comment by ItayZiv** [comment]
A font probably should be added to the style-guide.
Both Helvetica and Verdana seem to work. (The font needs to be available as the svg seems to render with the system fonts and not web fonts).

**Pull request merged**